react/src/components/AutoScalingRuleEditorModal.tsx

```diff
@@ -0,0 +1,375 @@
+                cooldown_seconds: values.cooldown_seconds,
+                min_replicas: values.min_replicas,
+                max_replicas: values.max_replicas,
+              },
+            },
+            onCompleted: (res, errors) => {
+              if (!res?.modify_endpoint_auto_scaling_rule_node?.ok) {
+                message.error(res?.modify_endpoint_auto_scaling_rule_node?.msg);
+                onRequestClose(false);
+                return;
+              }
+              if (errors && errors.length > 0) {
+                const errorMsgList = _.map(errors, (error) => error.message);
+                for (const error of errorMsgList) {
+                  message.error(error, 2.5);
+                }
+                onRequestClose(false);
+                return;
+              }
+              message.success(t('autoScalingRule.SuccessfullyUpdated'));
+              onRequestClose(true);
+            },
+            onError: (error) => {
+              message.error(error.message);
+              onRequestClose(false);
+            },
+          });
+        } else {
+          commitAddAutoScalingRule({
+            variables: {
+              endpoint: endpoint_id ?? '',
+              props: {
+                metric_name: values.metric_name,
+                metric_source: values.metric_source as AutoScalingMetricSource,
+                threshold: values.threshold,
+                comparator: values.comparator,
+                step_size: values.step_size,
+                cooldown_seconds: values.cooldown_seconds,
+                min_replicas: values.min_replicas,
+                max_replicas: values.max_replicas,
+              },
+            },
+            onCompleted: (res, errors) => {
+              if (!res?.create_endpoint_auto_scaling_rule_node?.ok) {
+                message.error(res?.create_endpoint_auto_scaling_rule_node?.msg);
+                onRequestClose(false);
+                return;
+              }
+              if (errors && errors.length > 0) {
+                const errorMsgList = _.map(errors, (error) => error.message);
+                for (const error of errorMsgList) {
+                  message.error(error, 2.5);
+                }
+                onRequestClose(false);
+                return;
+              }
+              message.success(t('autoScalingRule.SuccessfullyCreated'));
+              onRequestClose(true);
+            },
+            onError: (error) => {
+              message.error(error.message);
+              onRequestClose(false);
+            },
+          });
+        }
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  const handleCancel = () => {
+    onRequestClose(false);
+  };
+
+  const comparators = [
+    { label: '<', value: 'LESS_THAN' },
+    { label: '<=', value: 'LESS_THAN_OR_EQUAL' },
+    { label: '>', value: 'GREATER_THAN' },
+    { label: '>=', value: 'GREATER_THAN_OR_EQUAL' },
```

Comment:
How about using ≥ and ≤ ?
```suggestion
    { label: '≤', value: 'LESS_THAN_OR_EQUAL' },
    { label: '>', value: 'GREATER_THAN' },
    { label: '≥', value: 'GREATER_THAN_OR_EQUAL' },
```